_fixtures/examinememory_expr.go

```diff
@@ -0,0 +1,33 @@
+package main
```

Comment:
I would prefer if we could avoid creating a new fixture for this. See testvariables2.go.

Reply:
👍 sure, on first glance this file looks like it should cover enough cases. But if it doesn't can I modify it?

Reply:
Yes.

Reply:
Done . Got rid of this and used `testvariables2.go` instead.